contextily/tile.py

```diff
@@ -32,6 +39,23 @@
 memory = _Memory(tmpdir, verbose=0)
 
 
+def set_cache_dir(path):
+    """
+    Set a cache directory to use in the current python session.
+
+    By default, contextily caches downloaded tiles per python session, but
+    will afterwards delete the cache directory. By setting it to a custom
+    path, you can avoid this, and re-use the same cache a next time by
+    again setting the cache dir to that directory.
+
+    Parameters
+    ----------
+    path : str
+        Path to the cache directory.
+    """
+    memory.store_backend.location = path
+
+
 def _clear_cache():
```

Comment:
Could we make this method integrated with `set_cache_dir`? For example but setting `tmpdir` to `path` automatically?

Reply:
Not in the current design, as we want to delete the default tmpdir at the end, but the cache dir set with `set_cache_dir` does not need to be removed at the end of a session